src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/entity/ManaCreeper.java

```diff
@@ -24,12 +18,8 @@ public static AttributeSupplier.Builder createAttributes() {
 
     @Override
     protected void registerGoals() {
-        goalSelector.addGoal(1, new FloatGoal(this));
-        goalSelector.addGoal(5, new WaterAvoidingRandomStrollGoal(this, 0.8D));
-        goalSelector.addGoal(6, new LookAtPlayerGoal(this, Player.class, 8F));
-        goalSelector.addGoal(6, new RandomLookAroundGoal(this));
-        targetSelector.addGoal(1, new NearestAttackableTargetGoal<>(this, Player.class, true));
-        targetSelector.addGoal(2, new HurtByTargetGoal(this));
+        super.registerGoals();
+        goalSelector.getAvailableGoals().stream().filter(goal -> goal.getGoal() instanceof AvoidEntityGoal<?>).forEach(goal -> goalSelector.removeGoal(goal.getGoal()));
```

Comment:
What exactly is the purpose of this?

Reply:
de duplication of code (those goals are all defined by the creeper so we just let it behave like a normal creeper (that is not afraid of cats) until it is actually implemented

Reply:
why remove being afraid of cats tho? we could just remove the override as a whole and have it behave just like a normal creeper (with the extra mana vortex death effect)

Reply:
in the original the explosion did no physical damage only mana also this is not the PR to address this the changes would need to be made on 1.18 first and then merged

Reply:
hm. ok